packages/bpk-component-calendar/src/BpkCalendarDate.js

```diff
@@ -153,6 +157,7 @@ export const propTypes = {
   onClick: PropTypes.func,
   onDateKeyDown: PropTypes.func,
   preventKeyboardFocus: PropTypes.bool,
+  className: PropTypes.string,
```

Comment:
could you kindly put these in alphabetical order? (grouped by optional / required)

Reply:
Done!